protractor.conf.js

```diff
@@ -2,24 +2,32 @@ exports.config = {
 
     baseUrl: 'http://todomvc.com',
 
+    seleniumAddress: 'http://localhost:4444/wd/hub/',
```

Comment:
I agree, but without starting it manually the `--highlightDelay` won't work because the `blockingProxy` uses the `seleniumAddress`, unless you see an other solution.

I you don't favour the `highlightDelay` I can remove it and only keep the dependency upgrade as part of the PR.